### ISSUE TITLE
Support

### ISSUE THREAD

**Issue by j-calvert** [moom0o/PhotoSphereStudio#6]
Readme says

> If you have any questions about how to set this up or about the source code, feel free to ask either moom0o (moo#0529 on Discord) or WinsDominoes (Win#7206 on Discord).

On what server?  Or add as friends first?

**Comment by j-calvert** [comment]
In the meantime, my actual (non-meta) question is Re: Setting up local instance.  I think I've got all the credentials and config edits in place.  How do I initiate the local server behind http://localhost:7000 .  The most obvious thing doesn't seem to work.

```
jcalvert@Jeremys-MacBook-Pro PhotoSphereStudio % npm run
Lifecycle scripts included in photospherestudio@1.0.0:
  test
    echo "Error: no test specified" && exit 1


```

**Comment by j-calvert** [comment]
I've got it running locally.

I'm quite sure my photos do have location data (in standard EXIF metadata place).  Some of them are even panos (not photospheres) made with my Pixel 5a. But it's not getting parsed out.

Looking at the code, I'm guessing the API leaves one to guess at how the geo info is parsed out of the image file metadata?  Looking at the documentation now.
